ecs-cli/modules/commands/local/local_command.go

```diff
@@ -194,6 +203,11 @@ func flagDescription(longName, cmdName string) string {
 			psCmdName:   "Lists all locally running containers.",
 			downCmdName: "Stops and removes all locally running containers.",
 		},
+		flags.ForceFlag: {
+			// also does not prompt for override?
```

Comment:
Should we remove this comment?

Also should we shorten the flag description to something like `"Overwrite docker-compose.ecs-local.yml if it exists"`?